README.md

```diff
@@ -577,6 +577,49 @@ In case the database URL is subject to change, the `slug` parameter can use to i
             app
 ```
 
+### `ariga/atlas-action/setup`
+
+This action build the binary of atlas-action on your pipeline, instead of downloading it from the internet. So you can pin it and other actions to specified commit.
```

Comment:
```suggestion
This action builds the binary of atlas-action on your pipeline, instead of downloading it from the internet. So you can pin it and other actions to specified commit.
```